### PR TITLE
Cleanup some text strings files to return object literals

### PR DESCRIPTION
**Pull request by kialam**
##### SUMMARY

Small cleanup of help text files.

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

 - UI

### CODE REVIEW

File: awx/ui/src/screens/Project/shared/ProjectSubForms/SharedFields.js [hunk 1]
```diff
@@ -23,16 +23,13 @@ export const UrlFormField = ({ tooltip }) => (
...
-  return (
-    <FormField
-      id="project-scm-branch"
-      name="scm_branch"
-      type="text"
-      label={label}
-      tooltip={projectHelpStrings.branchFormField}
-    />
-  );
+  <FormField
```
Comment (keithjgrant) [inline awx/ui/src/screens/Project/shared/ProjectSubForms/SharedFields.js:26]: missing `return` (or change `{}` to `()` for function body)

**Comment by keithjgrant** [comment]
this cleanup make me happy 😄

**Review by nixocio — APPROVED** [review]
@kialam I like the consistency. 🥇 

I triggered the e2e tests.

**Review by InnocentK — APPROVED** [review]
The above failing test is unrelated to the work in this PR. Looks good from QE so merging

**Pull request merged**